src/html.rs

```diff
@@ -1,25 +1,15 @@
 use chrono::{DateTime, Utc};
-use fomat_macros::fomat;
-use std::cmp::Ordering;
-use std::collections::{btree_map, BTreeSet};
+use serde_json::value::{from_value, to_value, Value};
+use std::collections::HashMap;
```

Comment:
nit: we use FxHashMap everywhere so do the same here.
Anyway if you see any good reason to use a collections::HashMap here and in other places, please do that in an other patch.

Reply:
There's #283 which we filed a while ago, it'd be nice to finally fix it (switch everything to HashMap and drop FxHashMap).

Reply:
(I agree, in a separate patch)

Reply:
tera traits use HashMap.

Reply:
I saw that: https://docs.rs/tera/1.5.0/tera/trait.Function.html